asylum-connect/control-panel
# Updated service coverage dialog

## Description

~Added typeahead behavior to the CoverageEdit dialog. I've solicited some feedback in the Asana ticket, so I'm marking this as a "Draft" PR for now, pending a response.~

This PR adds select boxes to the CoverageEdit dialog for picking Country, State, City, and County, plus a textbox for Towns. It also adds some static data files with lists states and cities by state for Mexico, Canada, and the US. 

- Asana ticket: https://app.asana.com/0/1132189118126148/1180818950425345

## How to Test
1. Go to the edit page for an organization or service
2. Edit the ServiceAreaCoverage using the new component
3. Verify that it saves correctly to the org or service
